### PR TITLE
NXP-28448: run functional tests using nuxeo-ftest

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

**Comment by Gabez0r** [comment]
We're running tests from the ftest folder (see https://github.com/nuxeo/nuxeo-web-ui/pull/812/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R198).

File: Jenkinsfile [hunk 1]
```diff
@@ -120,16 +123,49 @@ pipeline {
...
+            Run Nuxeo Web UI Functional Tests
+            --------------------------"""
+            sh 'mvn -B -nsu -f plugin/itests/addon install'
+            sh 'mvn -B -nsu -f plugin/itests/marketplace install'
+            try {
+              sh 'mvn -B -nsu -f ftest install'
+            } finally {
+              try {
+                archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/*,**/log/*.log, **/target/cucumber-reports/*.json, **/nxserver/config/distribution.properties, **/failsafe-reports/*, **/target/results/*.html, **/target/screenshots/*.png, plugin/web-ui/marketplace/target/nuxeo-web-ui-marketplace-*.zip, plugin/itests/marketplace/target/nuxeo-web-ui-marketplace-*.zip, plugin/metrics/target/report/*'
+                cucumber fileIncludePattern: '**/*.json', jsonReportDirectory: 'ftest/target/cucumber-reports/', sortingMethod: 'NATURAL'
```
Comment (nmpcunha) [inline Jenkinsfile:149]: We also have cucumber reports for the addons that have functional tests.
Should we try to merge them too?

Comment (nmpcunha) [reply Jenkinsfile:149]: @nelsonsilva

Comment (nelsonsilva) [reply Jenkinsfile:174]: Made changes in https://github.com/nuxeo/nuxeo-web-ui/pull/812/commits/fa98b96fa9133156481ab1394df8008900224c50 to generate all the reports in the same folder 🤞

**Pull request merged**